core/common/src/geometry/FrustumPlanes.ts

```diff
@@ -129,22 +203,17 @@ export class FrustumPlanes {
   }
 }
 
-/** @internal */
+/** @public @extensions */
 export namespace FrustumPlanes { // eslint-disable-line no-redeclare
```

Comment:
side note, do we really need this namespace?

Reply:
The enum it contains is in use. I'm not looking to make breaking changes.

Reply:
Should we deprecate instead of promoting this to public and provide an alternative? Most usage is internal (or was before this pr)

Reply:
I was initially thinking since this is a static you could move the enum into it, but then type would not be referenceable; now seeing the value in using the namespace